api/app/tests/requests/test_pteams.py

```diff
@@ -2831,3 +2832,428 @@ def test_returns_summary_even_if_multiple_services_are_registrered(self):
+        ) -> dict:
+            url = f"/pteams/{pteam_id}/services/{service_id}/ticketstatus/{ticket_id}"
+            user1_access_token = self._get_access_token(USER1)
+            _headers = {
+                "Authorization": f"Bearer {user1_access_token}",
+                "Content-Type": "application/json",
+                "accept": "application/json",
+            }
+            return client.post(url, headers=_headers, json=request).json()
+
+        def _get_ticket_status(self, pteam_id: str, service_id: str, ticket_id: str) -> dict:
+            url = f"/pteams/{pteam_id}/services/{service_id}/ticketstatus/{ticket_id}"
+            user1_access_token = self._get_access_token(USER1)
+            _headers = {
+                "Authorization": f"Bearer {user1_access_token}",
+                "Content-Type": "application/json",
+                "accept": "application/json",
+            }
+            return client.get(url, headers=_headers).json()
+
+    class TestGet(Common):
+
+        def test_returns_default_if_no_status_created(self, testdb):
+            create_user(USER1)
+            pteam1 = create_pteam(USER1, PTEAM1)
+            tag1 = create_tag(USER1, TAG1)
+            # add test_service to pteam1
+            test_service = "test_service"
+            test_target = "test target"
+            test_version = "1.2.3"
+            refs0 = {tag1.tag_name: [(test_target, test_version)]}
+            upload_pteam_tags(USER1, pteam1.pteam_id, test_service, refs0)
+            service_id1 = self._get_service_id_by_service_name(USER1, pteam1.pteam_id, test_service)
+            # create actionable topic1
+            action1 = self._gen_action([tag1.tag_name])
+            topic1 = create_topic(USER1, {**TOPIC1, "tags": [tag1.tag_name], "actions": [action1]})
+            tickets = self._get_tickets(pteam1.pteam_id, service_id1, topic1.topic_id)
+            ticket_id1 = tickets[0]["ticket_id"]
+
+            url = f"/pteams/{pteam1.pteam_id}/services/{service_id1}/ticketstatus/{ticket_id1}"
+            user1_access_token = self._get_access_token(USER1)
+            _headers = {
+                "Authorization": f"Bearer {user1_access_token}",
+                "Content-Type": "application/json",
+                "accept": "application/json",
+            }
+            response = client.get(url, headers=_headers)
+            assert response.status_code == 200
+
+            data = response.json()
+            expected_status = {
+                "status_id": None,
+                "ticket_id": ticket_id1,
+                "topic_status": models.TopicStatusType.alerted.value,
+                "user_id": None,
+                "created_at": None,
+                "assignees": [],
+                "note": None,
+                "scheduled_at": None,
+                "action_logs": [],
+            }
+            assert data == expected_status
+
+        def test_returns_current_status(self):
```

Comment:
確かめたいことは、ユーザが入力したデータが反映されていること？
```suggestion
        def test_it_should_returns_current_status_if_user_updated(self):
```